JavaAssignments/src/mayur/SwapNumberWithoutThirdVeriable.java

```diff
@@ -0,0 +1,21 @@
+package mayur;
+
+public class SwapNumberWithoutThirdVeriable 
+{
+	public static void main(String[] args) 
+	{
+		float Num1 = (float) Double.parseDouble(args[0]);
```

Comment:
Please use Float.parseFloat(args[0]).
Approving your request.

Reply:
rest Code looks good to me.

Reply:
Used Float.parseFloat(args[0]) in other programs